docs/source/using/experimental-features.adoc

```diff
@@ -324,3 +324,23 @@ the following command:
 $ minishift config set auto-start-tray false
 ----
 
+[[timezone]]
+== Timezone Setup
+
+If you want to set a different timezone from the default one, use the following command:
+
+----
+$ minishift timezone --set <Valid_Timezome>
```

Comment:
The docs changes are still there, but the `timezone` command is hidden and we have not yet decided if we are going to keep that command in the future.

Reply:
there are in the experimental topic

Reply:
Right... we can change this later, but an example using the start flag `minishift start --timezone`  would have been more apt.